src/config/cfg_rules.ini

```diff
@@ -698,3 +699,17 @@ validator = ini_allowed_options
 section_re = ^application/.*$
 
 option = inherit_from
+
+[rule/allowed_subdomain_options]
+validator = ini_allowed_options
+section_re = ^domain/[0-9A-Za-z.]\+/[0-9A-Za-z.]\+$
```

Comment:
I think that application domains should be able to use subdomain condiguration as well.
If it is not allowed then it is a bug and need to be fixed.